qt/scientific_interfaces/Inelastic/Analysis/ConvFitAddWorkspaceDialog.cpp

```diff
@@ -73,7 +73,7 @@ ConvFitAddWorkspaceDialog::ConvFitAddWorkspaceDialog(QWidget *parent) : QDialog(
 
   connect(m_uiForm.dsWorkspace, SIGNAL(dataReady(const QString &)), this, SLOT(workspaceChanged(const QString &)));
   connect(m_uiForm.ckAllSpectra, SIGNAL(stateChanged(int)), this, SLOT(selectAllSpectra(int)));
-  connect(m_uiForm.pbAdd, SIGNAL(clicked()), this, SIGNAL(addData()));
+  connect(m_uiForm.pbAdd, SIGNAL(clicked()), this, SLOT(emitAddData()));
```

Comment:
Why you added custom signals?

Reply:
I needed to change this signal to a slot because I wanted to pass an argument to the `addData` signal. I'm not aware of another way of doing this? Apart from when the previous signal has a matching signature e.g.
```
connect(m_uiForm.ckAllSpectra, SIGNAL(stateChanged(int)), this, SIGNAL(selectAllSpectra(int)));
```